[BUG] panic: runtime error: invalid memory address or nil pointer dereference

**Describe the bug**
The bug appeared only on the version 2.0.0. On the version 1.6.0, it didn't appear. If there is a variable without default value, regula will crash.

**Operating System / Tools**
* OS: Ubuntu 18.04.6 or Mac OS

**How you're running Regula**
I'm using Regula v2.0.0 as a Rego library with OPA v0.34.1 

```shell
$ regula -v run tf
DEBUG Could not find configuration file: Config File ".regula.yaml" Not Found in "[/Users/loginOne/src/tmp/regula_tt /Users/loginOne/src/tmp /Users/loginOne/src /Users/loginOne /Users /]"
DEBUG local.s3_cors_origins: terraform.workspace -> cty.StringVal("terraform.workspace")
DEBUG local.s3_cors_origins: var.dashboard_url -> variable.dashboard_url
DEBUG evaluate: variable.dashboard_url
DEBUG evaluate: local.s3_cors_origins
DEBUG local.s3_cors_origins: terraform.workspace -> cty.StringVal("terraform.workspace")
DEBUG local.s3_cors_origins: var.dashboard_url -> variable.dashboard_url
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x13ae1a8]

goroutine 1 [running]:
github.com/zclconf/go-cty/cty.Type.FriendlyName(...)
	/home/runner/go/pkg/mod/github.com/zclconf/go-cty@v1.8.4/cty/type.go:47
github.com/hashicorp/hcl/v2/hclsyntax.(*ConditionalExpr).Value(0xc00024ce00, 0xc000411260)
	/home/runner/go/pkg/mod/github.com/hashicorp/hcl/v2@v2.10.0/hclsyntax/expression.go:590 +0x1348
github.com/hashicorp/hcl/v2/hclsyntax.(*TupleConsExpr).Value(0xc00031b360, 0xc00079c180)
	/home/runner/go/pkg/mod/github.com/hashicorp/hcl/v2@v2.10.0/hclsyntax/expression.go:744 +0x107
github.com/fugue/regula/pkg/regulatf.(*Evaluation).evaluate(0xc000455740)
	/home/runner/work/regula/regula/pkg/regulatf/regulatf.go:222 +0x7a9
github.com/fugue/regula/pkg/regulatf.EvaluateAnalysis(0xc000410c60)
	/home/runner/work/regula/regula/pkg/regulatf/regulatf.go:158 +0x151
github.com/fugue/regula/pkg/loader.newHclConfiguration(0xc00040ae80)
	/home/runner/work/regula/regula/pkg/loader/tf.go:103 +0x2a
github.com/fugue/regula/pkg/loader.(*TfDetector).DetectDirectory(0xc0003b2100, {0x1d658c0, 0xc00001a840}, {0x50, 0x20})
	/home/runner/work/regula/regula/pkg/loader/tf.go:93 +0x185
github.com/fugue/regula/pkg/loader.(*directory).DetectType(0xc00040ad60, {0x1d52210, 0x252b068}, {0x70, 0xad})
	/home/runner/work/regula/regula/pkg/loader/input.go:32 +0x34
github.com/fugue/regula/pkg/loader.(*AutoDetector).DetectDirectory(0xc000821bf0, {0x1d658c0, 0xc00001a840}, {0x20, 0x85})
	/home/runner/work/regula/regula/pkg/loader/auto.go:26 +0xbc
github.com/fugue/regula/pkg/loader.(*directory).DetectType(0x7ffeefbfed96, {0x1d52198, 0xc000410b40}, {0x1, 0x0})
	/home/runner/work/regula/regula/pkg/loader/input.go:32 +0x34
github.com/fugue/regula/pkg/loader.LocalConfigurationLoader.func1()
	/home/runner/work/regula/regula/pkg/loader/loadpaths.go:124 +0x5b0
github.com/fugue/regula/cmd.NewRunCommand.func1(0xc0003bec80, {0xc00040abc0, 0x1, 0x2})
	/home/runner/work/regula/regula/cmd/run.go:144 +0x777
github.com/spf13/cobra.(*Command).execute(0xc0003bec80, {0xc00040aba0, 0x2, 0x2})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:856 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x24c5ac0)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:902
github.com/fugue/regula/cmd.Execute()
	/home/runner/work/regula/regula/cmd/root.go:47 +0x25
main.main()
	/home/runner/work/regula/regula/main.go:22 +0x17
```

**IaC Configuration**
```hcl
# KO
variable "dashboard_url" {}

# OK <----- Uncomment this one and comment the one above
#variable "dashboard_url" {
#  default = "http://example.com"
#}

locals {
  s3_cors_origins = [terraform.workspace == "dev" ? "*" : var.dashboard_url]
}

```

--- a/pkg/loader/tf_test/issue-245.json
+++ b/pkg/loader/tf_test/issue-245.json
@@ -1,0 +1,7 @@
+{
+  "content": {
+    "hcl_resource_view_version": "0.0.1",
+    "resources": {}
+  },
+  "filepath": "tf_test/issue-245"
+}

--- a/pkg/loader/tf_test/issue-245/test.tf
+++ b/pkg/loader/tf_test/issue-245/test.tf
@@ -1,0 +1,11 @@
+# KO
+variable "dashboard_url" {}
+
+# OK <----- Uncomment this one and comment the one above
+#variable "dashboard_url" {
+#  default = "http://example.com"
+#}
+
+locals {
+  s3_cors_origins = [terraform.workspace == "dev" ? "*" : var.dashboard_url]
+}

--- a/pkg/regulatf/moduletree.go
+++ b/pkg/regulatf/moduletree.go
@@ -225,11 +225,13 @@ func walkModule(v Visitor, moduleName ModuleName, module *configs.Module) {
 	name := EmptyFullName(moduleName)
 
 	for _, variable := range module.Variables {
-		expr := hclsyntax.LiteralValueExpr{
-			Val:      variable.Default,
-			SrcRange: variable.DeclRange,
+		if variable.Default != cty.NilVal {
+			expr := hclsyntax.LiteralValueExpr{
+				Val:      variable.Default,
+				SrcRange: variable.DeclRange,
+			}
+			v.VisitExpr(name.AddKey("variable").AddKey(variable.Name), &expr)
 		}
-		v.VisitExpr(name.AddKey("variable").AddKey(variable.Name), &expr)
 	}
 
 	for _, local := range module.Locals {

